code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -17,15 +17,15 @@
 
 /obj/item/weapon/gun/energy/laser/practice
 	name = "NT LG \"Lightfall\" - P"
-	desc = "A modified version of the NeoTheology brand laser carbine, this one fires less concentrated energy bolts designed for target practice."
+	desc = "A modified version of \"NeoTheology\" brand laser carbine, this one fires less concentrated energy bolts, designed for target practice."
 	matter = list(MATERIAL_PLASTEEL = 20, MATERIAL_WOOD = 8, MATERIAL_SILVER = 2)
 	projectile_type = /obj/item/projectile/beam/practice
 
 obj/item/weapon/gun/energy/retro
 	name = "Alliance LG \"Cog\""
 	icon_state = "retro"
 	item_state = "retro"
-	desc = "Alliance cheap laser gun. In our deep past - this was main weapon of low-rank police force across all Milky Way, bilions copys of this gun was made. They still can be found almoust everywhere."
```

Comment:
Improved, but not quite right. Here's a better version

 "Alliance cheap laser gun. In the distant past - this was the main weapon of low-rank police forces across all the Milky Way, billions of copies of this gun were made. They are ubiquitous."

Reply:
Not quite right. should be "were made"